plugins/community/neoeinstein-prost/v0.3.1/buf.plugin.yaml

```diff
@@ -16,6 +16,7 @@ registry:
       - name: "prost"
         req: "0.12.3"
         default_features: false
+        features: ["std"]
```

Comment:
Is this difference between this and setting `default_features: true` that it would enable the `derive` feature too?

Reply:
yeah, that's the difference. Not sure if we want to enable `derive` or not. My vague understanding (from the [README](https://docs.rs/prost/0.12.3/prost/index.html#serializing-existing-types)) is that it's more for user-generated types, but OTOH the custom `derive` macro is added to the generated code.

Let's maybe just add it (by using `default_features: true`) — it's a single extra crate with no dependencies.